index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Tic Tac Toe</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
```

Comment:
This is incorrect I believe. We don't have that file in the repo yet, so it will result in a 404 error. Please remove. Styles are out of scope for this task anyway.